packages/auth/amplify_auth_cognito_dart/lib/src/auth_plugin_impl.dart

```diff
@@ -98,39 +81,40 @@ class AmplifyAuthCognitoDart extends AuthPluginInterface<
       CognitoUserAttributeKey,
       AuthUserAttribute<CognitoUserAttributeKey>,
       CognitoDevice,
-      CognitoSignUpOptions,
```

Comment:
So when are not going to strongly type `pluginOptions` when using `Auth.getPlugin(AuthPluginKey)`?

Reply:
Nope